src/Discussion/DiscussionPolicy.php

```diff
@@ -29,25 +28,18 @@ class DiscussionPolicy extends AbstractPolicy
      */
     protected $settings;
 
-    /**
-     * @var Gate
-     */
-    protected $gate;
-
     /**
      * @var Dispatcher
      */
     protected $events;
 
     /**
      * @param SettingsRepositoryInterface $settings
-     * @param Gate $gate
      * @param Dispatcher $events
      */
-    public function __construct(SettingsRepositoryInterface $settings, Gate $gate, Dispatcher $events)
+    public function __construct(SettingsRepositoryInterface $settings, Dispatcher $events)
```

Comment:
This was just a forgotten dependency, apparently?

Reply:
yep, probably from when we started using $actor->can instead.